### DISCUSSION TITLE
Add support for inline rolls in formulas

### DISCUSSION THREAD

**Discussion by federicoacker** [MeusRex/pf2e-chex#11] [Ideas]
It would be a nice feature to have to be able to use inline rolls for the formulas for things like improvements (or terrain in a future feature). This would solve the problem of how to randomize resources on tiles

**Comment by MeusRex** [comment]
hmm, income: stone: r/1d4
Like that? Shouldn't be too hard to do. But only for income, it would be a bit odd for travel :D

**Reply by federicoacker** [reply]
Oh something like with foundry's native inline rolls so something like income: ore: [[/r 1d4+2]] for example

**Reply by federicoacker** [reply]
So that people don't have to learn a new roll command and can just use the ones from foundry xD

**Reply by MeusRex** [reply]
Makes sense, and there has to be some kind of roll-command parser somewhere in the API that I can re-use for this.

**Reply by federicoacker** [reply]
Yeaaah I think so, maybe there's some way to make it so that users can just use foundry's macro language in it, like make it so that the formula just launches a macro
